index.bs

```diff
@@ -248,12 +250,12 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
   </section>
   <section>
     <h3 id="container-policies">Container policies</h3>
-    <p>In addition to the <a>header policy</a>, each [=nested browsing context=]
-    has a <dfn export>container policy</dfn>, which is a <a>policy directive</a>,
+    <p>In addition to the <a>header policy</a>, each [=nested navigable=] has a
```

Comment:
I think this and other references should be child navigable, not nested navigable. nested navigable is a property; child navigable is a data type.

Reply:
That's exactly the sort of thing I expected to mess up -- thanks!

Reply:
Just opened https://github.com/whatwg/html/issues/8713 because this is too easy to mess up.